src/pages/Dashboard/components/Calendar/CalendarContainer.tsx

```diff
@@ -97,39 +88,27 @@ const Calendar = ({ assignations }: CalendarProps): JSX.Element => {
 
   return (
     <>
-      <CalendarContainer>
-        <CalendarHeader>
-          <CalendarHeaderPrev onClick={() => setValue(prevMonth())}>{String.fromCharCode(171)}</CalendarHeaderPrev>
-          <div>
-            {currMonthName()} {currYear()}
-          </div>
-          <CalendarHeaderNext onClick={() => setValue(nextMonth())}>{String.fromCharCode(187)}</CalendarHeaderNext>
-        </CalendarHeader>
-        <Body>
-          <DayNames>
-            {daysNames.map((d, i) => (
-              <Week key={i}>{d}</Week>
-            ))}
-          </DayNames>
-          {calendar.map((week, i) => (
-            <div key={i}>
-              {week.map((day: Moment, i) => (
-                <Day key={i} onClick={() => displayTasks(day)}>
-                  <DayDiv isToday={isToday(day)} beforeToday={beforeToday(day)} isSelected={isSelected(day)}>
-                    {day.format('D').toString()}
-                  </DayDiv>
-                  <DayDivTask hasEvent={hasEvent(day)} />
-                </Day>
-              ))}
-            </div>
-          ))}
-        </Body>
-      </CalendarContainer>
+      <Container>
```

Comment:
Could you please change to CalendarContainerStyled  as when imported it can be confused with the <Continaer/> component from MUI ?

Reply:
ok